README.md

```diff
@@ -4,3 +4,12 @@ Docker image for sending daily reports via email
 ## Purpose
 
 This is intended to be used by xPub (https://github.com/elifesciences/elife-xpub) and by Libero Reviewer for generating database reports on a (bi-)daily basis.
+
+## Examples
+
+`make build` - This will build the docker container packaging the scripts that are run can be found in `cron_tasks_folder`.
```

Comment:
This folder is not copied in the container image by the Dockerfile, but I think that is intended as otherwise they would have to be in the `elife-xpub-formula`? No preference either way so far.

Reply:
The intention was as much as possible remains configurable at deployment time - rather than at container build time.

Reply:
Just looking at the formula ... maybe we should bake these scripts into the image. As I'm now thinking that it makes more sense to have the scripts sat near the code (for the schema dependent reasons above) rather than near deployment. :thinking:  ... does that tip the balance for you @giorgiosironi

Reply:
I agree on baking the scripts in the image because despite the hardcoding of configuration such as the `8` and `13` magic values, it reduces the duplication of having them here to be tested and in the formula to be deployed; also, it makes the lifecycle clear, pull requests will be opened against this repository.

Reply:
Ok - I've done it - makes more sense!